appcenter/AppCenter.d.ts

```diff
@@ -0,0 +1,30 @@
+export const enum AppCenterLogLevel {
+    LogLevelVerbose = 2,
```

Comment:
Using pascal case for enums in our internal code was an inconsistency with Crashes.

In the typescript version since we are defining new const we should take the opportunity to align.

In other APIs we are adding we are planning to use THIS_CONVENTION.

Let's change this enum to using the following values without the LogLevel prefix as we are already in the enum:

```
   VERBOSE = 2,
   DEBUG = 3,
   // etc...
```

Reply:
We should also reference `AppCenterLogLevel.LogLevelDebug` instead of using the integer value directly here, like you did for constants at top level.

Reply:
Constants at top level referenced the Typescript implementation (integers). They had to be changed to new CAPS method. I'm not sure we can reference a JS enum in a definition file, tried some ways but didn't work as it has to be a constant expression, any suggestions?

Reply:
It's fine then, we can keep integers for now.